.github/workflows/build.yml

```diff
@@ -275,22 +264,74 @@ jobs:
           git config --global core.symlinks true
           git config --global user.email 'renovate@whitesourcesoftware.com'
           git config --global user.name  'Renovate Bot'
-          git --version
-          echo "Node $(node --version)"
-          echo "Yarn $(yarn --version)"
 
       - name: Cache jest
         uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # v3.3.1
         with:
           path: .cache/jest
-          key: jest-cache-${{ runner.os }}-${{ env.NODE_VERSION }}-${{ hashFiles('yarn.lock') }}
+          key: |
+            jest-cache-${{
+              runner.os
+            }}-${{
+              env.NODE_VERSION
+            }}-${{
+              hashFiles('yarn.lock')
+            }}-${{
+              matrix.cache-key
+            }}
 
       - name: Unit tests
-        run: yarn jest --ci --coverage ${{ env.coverage }}
+        shell: bash
+        run: |
+          for shard in ${{ matrix.shards }};
```

Comment:
`${{ matrix.shards }}` is empty

![image](https://github.com/renovatebot/renovate/assets/1798109/7a09d156-3b6c-4e36-afd3-a6e67fbafa1b)